resources/release-notes.md

```diff
@@ -10,6 +10,34 @@ description: |
 body="The version numbers used in headers on this page refers to the version of
 this very documentation, not to a version of any APIs described by it." %}
 
+
+## 11 Nov 2020
+
+### Version 2.0.0
+
+:tada: Launched new developer portal theme: :tada:
+*   New design :sparkles:
+*   New menu structure :orange_book:
+*   Reading time :hourglass:
+
+Other changes:
+*   Added GDPR disclaimer.
+*   Added custom 404 page.
+*   Added Transaction on File section.
+*   Added Seamless View for [Trustly][trustly-payments].
+*   Added documentation about payment states in
+    [Card Other Features][card-transaction-states]
```

Comment:
Doesn't the payment state documentation apply to all APIs, not just Card?

Reply:
Yes it does. Is it better to leave it without a link? We added one link as an example.